app/controllers/verify/review_controller.rb

```diff
@@ -34,7 +34,7 @@ def create
     private
 
     def idv_profile_complete?
-      idv_session.resolution.try(:success?)
+      idv_session.profile_confirmation == true
```

Comment:
just a naming nit. The profile response is a `resolution` not a `confirmation`. That's the language that most of the vendors use. Resolving the contact info to a single record is step 1, then confirming financials and phone are steps 2 and 3.